src/index.ts

```diff
@@ -1,5 +1,27 @@
-export function helloWorld() {
-  console.log('hello likepay');
+import BigNumber from 'bignumber.js';
+import { v4 } from 'uuid';
+import * as QRCode from 'qrcode';
+
+import { getUserInfo } from './util/api';
+import { COSMOS_DENOM } from './constant';
+
+export async function createPaymentQRCode(selector: string, likerId: string, amount: number) {
+  const user = await getUserInfo(likerId);
+  const { cosomosWallet } = user;
+  const cosmosAmount = {
+    denom: COSMOS_DENOM,
+    amount: new BigNumber(amount).multipliedBy(1e9).toFixed(),
+  };
+  const uuid = v4();
+  const payload = JSON.stringify({
+    wallet: cosomosWallet,
+    amount: cosmosAmount,
+    memo: uuid,
```

Comment:
Why named `memo`? How about `paymentID`?

Reply:
because i want to name that like how it is named in cosmos tx, so the remarks field is called `memo` in a cosmos tx
but then the `wallet` and `amount` should be called
`address` and `coins`